starter/orchestrations/CounterOrchestration.cs

```diff
@@ -0,0 +1,27 @@
+using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.DurableTask;
+
+namespace Spike.Inventory
+{
+    public static class DfSpikeOrch
+    {
+        [FunctionName("DfSpikeOrch")]
```

Comment:
Maybe change the `FunctionName` to match the one in ch02: "Orchestrator"?

Reply:
Ok... Will do

Reply:
Renamed Class, Function attribute and Namespace. Drops the word `Spike`  from all places including project name.

New namespace - `InventoryManagement`